### PR TITLE
WFLY-16352 Conscious Language in Singleton subsystem

### PR DESCRIPTION
**Pull request by rhusar**
Resolves
https://issues.redhat.com/browse/WFLY-16352

### CODE REVIEW

**Comment by rhusar** [comment]
@pferraro Please check if the wording is sufficient. Thanks.

File: clustering/singleton/server/src/main/java/org/wildfly/clustering/singleton/server/SingletonLogger.java [hunk 1]
```diff
@@ -68,7 +68,7 @@ public interface SingletonLogger extends BasicLogger {
     void serviceStartFailed(@Cause StartException e, String service);
 
     @LogMessage(level = WARN)
-    @Message(id = 6, value = "Failed to reach quorum of %2$d for %1$s service. No singleton master will be elected.")
+    @Message(id = 6, value = "Failed to reach quorum of %2$d for %1$s service. No singleton provider will be elected.")
```
Comment (pferraro) [inline clustering/singleton/server/src/main/java/org/wildfly/clustering/singleton/server/SingletonLogger.java:71]: > No _primary_ singleton provider will be elected.

File: clustering/singleton/extension/src/main/resources/org/wildfly/extension/clustering/singleton/LocalDescriptions.properties [hunk 2]
```diff
@@ -18,7 +18,7 @@ singleton.election-policy.random.remove=Removes a random election policy
 singleton.election-policy.simple=A simple election policy
 singleton.election-policy.simple.add=A simple election policy
 singleton.election-policy.simple.remove=Removes a simple election policy
-singleton.election-policy.simple.position=The position in the membership list from which the singleton master will be chosen
+singleton.election-policy.simple.position=The position in the membership list from which the singleton provider will be chosen
```
Comment (pferraro) [inline clustering/singleton/extension/src/main/resources/org/wildfly/extension/clustering/singleton/LocalDescriptions.properties:21]: > The position in the membership list from which the _primary_ singleton provider will be chosen

**Review by pferraro — CHANGES_REQUESTED** [review]

File: clustering/singleton/extension/src/main/resources/org/wildfly/extension/clustering/singleton/LocalDescriptions.properties (hunk 2 above)
Comment (rhusar) [reply clustering/singleton/extension/src/main/resources/org/wildfly/extension/clustering/singleton/LocalDescriptions.properties]: Ooops

**Review by pferraro — APPROVED** [review]

**Pull request merged**